pkg/server/event_handler_test.go

```diff
@@ -468,12 +498,42 @@ func (p *successProcessor) Process(_ context.Context, m *structpb.Struct) error
 }
 
 // failMessenger Send always fail.
-type failMessenger struct{}
 
-func (m *failMessenger) Send(_ context.Context, _ *v1alpha1.PmapEvent) error {
+type testMessanger interface {
+	Messenger
+	getPmapEvent() *v1alpha1.PmapEvent
+}
```

Comment:
We don't need this interface as long as our fake messenger implements the `Messenger` interface.